src/redux/reducers/topicReducer.js

```diff
@@ -0,0 +1,17 @@
+import createReducer from './createReducer';
+import { getTopicsSuccess, getTopicsError } from 'redux/actions/topicAction';
+
+const intialState = {
+  values: undefined,
+  error: undefined
+};
+
+export default createReducer(intialState, {
+  [getTopicsSuccess]: (state, { payload: { topics } }) => ({
+    values: topics
+  }),
+  [getTopicsError]: (state, { payload }) => ({
+    ...state,
+    error: (payload && payload[0]) || 'An error occured. Please try again.'
```

Comment:
If you will use that 'An error occured. Please try again.' in every reducer, it could be a constant. TIOLI

Reply:
Yeah, it's weird. I recently create a #22 that tries to solve that issue